### ISSUE TITLE
Does the editor support Angular 8

### ISSUE THREAD

**Issue by vijaychebrolu** [microsoft/monaco-editor#1594]
We are working on Angular 8 web app in a .net core 2.2 solution and need your help to integrate Monaco editor in our app.
We tried to install the npm package as  detailed in https://www.npmjs.com/package/monaco-editor. But we are getting errors while installing the package.
Does the editor support angular 8 and if yes, any reference to samples plz.

**Comment by NothingEverHappens** [comment]
Yes, it should work, here are some examples of existing components wrapping Monaco for Angular:
https://github.com/atularen/ngx-monaco-editor#readme
https://github.com/courage007/Angular-Monaco-Editor

**Comment by pavangayakwad** [comment]
Tried both recommendations above but they are fragile and didn't work well with Angular CLI 12. 

Just don't waste time, use this: https://www.npmjs.com/package/@ng-util/monaco-editor 

Highly recommended!
